mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/type/impl/MongoDBTypeContributor.java

```diff
@@ -20,5 +20,6 @@
 	@Override
 	public void contribute(TypeContributions typeContributions, ServiceRegistry serviceRegistry) {
 		typeContributions.contributeType( new StringAsObjectIdType() );
+		typeContributions.contributeType( new BytesAsBsonBinaryType() );
```

Comment:
@DavideD . The change links with the PR. You can see to test BuiltinTypesTest. And ... You can remove my change and run the test.